R/error-checks.R

```diff
@@ -29,7 +29,11 @@ isSameLength <- function(...) {
 }
 
 
-validateIsOfType <- function(object, type) {
+validateIsOfType <- function(object, type, nullAllowed = FALSE) {

```

Comment:
I think it should not necessarily crash if the object is null. (for example for optional parameters)